File: Android/Martinshare/app/src/main/java/mobile/martinshare/com/martinshare/API/protocols/IsLoggedInProtocol.java
package mobile.martinshare.com.martinshare.API.protocols;

/**
 * Created by Modestas Valauskas on 03.06.2015.
 */
public interface IsLoggedInProtocol {
    public void startedChecking();
    public void isLoggedIn();
    public void isNotLoggedIn();
    public boolean emptyCredentials( String username, String key);
    public void neverWasLoggedIn();
}

File: iOS/Martinshare/Martinshare/IsLoggedInProtocol.swift
//
//  IsLoggedInProtocol.swift
//  Martinshare
//
//  Created by Modestas Valauskas on 25.05.15.
//  Copyright (c) 2015 ModestasV Studios. All rights reserved.
//

import Foundation

protocol IsLoggedInProtocol {

    func startedChecking()
    func isLoggedIn()
    func isNotLoggedIn()
    func emptyCredentials(_ goodname: String, key:String)->Bool
    func neverWasLoggedIn()
    
}
